cypress/e2e/signIn.cy.js

```diff
@@ -21,15 +21,11 @@ describe('Sign In page', () => {
     signInPage.visit();
     cy.register(user.email, user.username, user.password);
```

Comment:
```suggestion
    cy.login(user.email, user.username, user.password);
```
use cy login function. It register the user and set cookies.